apps/guide/src/middleware.ts

```diff
@@ -8,9 +8,16 @@ export function middleware(request: NextRequest) {
 		return NextResponse.redirect(new URL(newUrl.pathname, request.url));
 	}
 
-	return NextResponse.redirect(new URL('/legacy', request.url));
+	// Redirect old urls to /legacy
+	if (!request.nextUrl.pathname.startsWith('/legacy') && !request.nextUrl.pathname.startsWith('/voice')) {
+		const newUrl = request.nextUrl.clone();
+		newUrl.pathname = `/legacy${newUrl.pathname}`;
+		return NextResponse.redirect(new URL(newUrl.pathname, request.url));
```

Comment:
This should be enough (i know i was the one that introduced this above... 😅)
```suggestion
		return NextResponse.redirect(newUrl);
```